sandbox/modules/identity/build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
   implementation project(':sandbox:libs:opensearch-authn')
 
   implementation 'org.greenrobot:eventbus:3.2.0'
+  implementation 'com.google.guava:guava:31.1-jre'
```

Comment:
Guava is a forbidden API in opensearch: https://github.com/opensearch-project/OpenSearch/blob/main/gradle/forbidden-dependencies.gradle#L15-L17

Alternative data structures should be used. I'm not quite sure of the suggested replacements for ImmutableSet and ImmutableList.